x230t/NEWS

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
don't copy over the NEWS file. create a new one. we want to do seperate skulls-x230t releases.